Tests/SwiftStdlibTests/CodableExtensionsTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CodableExtensionsTests.swift
```

Comment:
You should be testing for `Decodable` only sense this is what you're extending in your extension